tests/unittest.cpp

```diff
@@ -3840,13 +3840,26 @@ TEST_CASE("task_timer")
         b = true;
     });
 
-    this_thread::sleep_for(3 * timer.get_tick_length());
+    asio::steady_timer t(io_service);
+    asio::error_code ec;
```

Comment:
```suggestion
    asio_error_code ec;
```

So this would still compile with Boost